roles/etcd/templates/etcd-rkt.service.j2

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=etcd rkt wrapper
+Documentation=https://github.com/coreos/etcd
+Wants=network.target
+
+[Service]
+Restart=on-failure
+RestartSec=10s
+TimeoutStartSec=0
+LimitNOFILE=40000
+
+ExecStart={{ rkt_bin_dir | default("/usr/bin") }}/rkt run \
+--uuid-file-save=/var/run/etcd.uuid \
+--volume=etc-ssl-certs,kind=host,source=/etc/ssl/certs,readOnly=true \
+--mount=volume=etc-ssl-certs,target=/etc/ssl/certs \
+--volume=etcd-cert-dir,kind=host,source={{ etcd_cert_dir }},readOnly=true \
+--mount=volume=etcd-cert-dir,target={{ etcd_cert_dir }} \
+--volume=var-lib-etcd,kind=host,source=/var/lib/etcd,readOnly=false \
+--mount=volume=var-lib-etcd,target=/var/lib/etcd \
+--set-env-file=/etc/etcd.env \
+--stage1-from-dir=stage1-fly.aci \
+{{ etcd_image_repo }}:{{ etcd_image_tag }} \
+--name={{ etcd_member_name | default("etcd") }}
+
+ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/run/etcd.uuid
```

Comment:
That's a good question. 
I modeled the unit file off of coreos documentation [1] which uses the above checks. If we lose the uuid file, I'm not sure there's a way aside from writing a script to scrape the output of `rkt list` to address it. Since it's using the `stage-1-fly` aci, it's sharing the host networking, so a secondary kubelet would fail to start because the existing/stale kubelet would be bound to the same port. 
I did a quick search to rkt issues and came across [2] which mentions `KillMode=control-group` so that may be something to investigate. Maybe @ant31 has some comments?

[1] https://coreos.com/kubernetes/docs/latest/deploy-master.html#create-the-kubelet-unit
[2] https://github.com/coreos/rkt/issues/3295